FirebaseRemoteConfig/Sources/RCNConfigFetch.m

```diff
@@ -191,7 +196,84 @@ - (void)fetchConfigWithExpirationDuration:(NSTimeInterval)expirationDuration
                                          withError:error];
     }
     strongSelf->_settings.isFetchInProgress = YES;
-    [strongSelf refreshInstallationsTokenWithCompletionHandler:completionHandler];
+    NSString *fetchTypeHeader = [NSString stringWithFormat:@"%@/1", kBaseFetchType];
+    [strongSelf refreshInstallationsTokenWithCompletionHandler:fetchTypeHeader
+                                             completionHandler:completionHandler];
+  });
+}
+
+- (void)realtimeFetchConfigWithExpirationDuration:(NSTimeInterval)expirationDuration
```

Comment:
does this method need `expirationDuration`? for realtime i think it will always be set to 0?

Reply:
That's true, it doesn't need it. Removed and updated naming.